e2e/tests/misc/sidenav.spec.ts

```diff
@@ -0,0 +1,107 @@
+import Button from '../../app/aou-elements/button';
+import {SideNavLink} from '../../app/authenticated-page';
+import GoogleLoginPage from '../../app/google-login';
+import ProfilePage from '../../app/profile-page';
+import WorkspacesPage from '../../app/workspaces-page';
+import launchBrowser from '../../driver/puppeteer-launch';
+import {waitForExists} from '../../driver/xpath-util';
+
+const configs = require('../../resources/workbench-config');
+
+export const HELP_DESK = {
+  ASK_QUESTION: 'Ask a question about the Researcher Workbench',
+  REPORT_DATA_PRIVACY_CONCERN: 'Report a data privacy concern',
+  TELL_US_ABOUT_PUBLICATION: 'Tell us about an upcoming publication',
+  REQUEST_ADDITIONAL_BILLING_CREDITS: 'Request additional billing credits',
+};
+
+jest.setTimeout(2 * 60 * 1000);
```

Comment:
Does this affect other test cases? Add a comment?

Reply:
Added this comment in all `.spec.ts` file. As new tests are added in future in same suite, timeout should be increased.
`// set timeout globally per suite, not per test.`